typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
 
 
 # get the directory path to this file; 
-LIBS_DIR = os.path.dirname(os.path.abspath(__file__))
-PLUGIN_DIR = os.path.dirname(LIBS_DIR)
-PACKAGES_DIR = os.path.dirname(PLUGIN_DIR)
+LIBS_DIR = dirname(dirname(os.path.abspath(__file__)))
+PLUGIN_DIR = dirname(LIBS_DIR)
+PACKAGES_DIR = dirname(PLUGIN_DIR)
```

Comment:
The reason is ``sublime.packages_path()`` returns nothing if run at importing time. And the vars are used in some places during importing time. Alternative solution is to initialize them during ``plugin_loaded`` function, which is a problem for sublime 2 because it is not called in sublime 2. Therefore I figure it would be easier if I can figure the path out during importing time. I'll explore a bit to see how python deal with symlink on linux and OS X, it looks it behaves differently from windows.